lib/Codegen/Brutus.cpp

```diff
@@ -1,18 +1,24 @@
 #include "brutus/brutus.h"
+#include "brutus/brutus_internal.h"
 #include "brutus/Dialect/Julia/JuliaOps.h"
+
 #include "mlir/InitAllDialects.h"
 
 extern "C" {
-jl_sym_t *call_sym;    jl_sym_t *invoke_sym;
```

Comment:
I thought this is necessary to anchor the extern variables in this compilation unit

Reply:
Oops, you're right, they still need to be defined somewhere